recipes/kinex/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "kinex" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/bedapub/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: 8f153d85657e16f0bae58df8d5a6cb52f8ba28ff992203e7762d81e04a448e0d
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps -vv
+  run_exports:
+    - {{ pin_subpackage("kinex", max_pin="x") }}
+
+requirements:
+  host:
+    - pip
+    - python
```

```suggestion
    - python >=3.8,<=3.11
```
Please pin the python version in `host` as well.